### PR TITLE
added another EEPROM MicroChip 24FC64: 64 kBit

### PR DESCRIPTION
**Pull request by hgrodriguez**

### CODE REVIEW

File: index/ee/eeprom_i2c/eeprom_i2c-4.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+licenses = "BSD-3-Clause"
+
+authors = ["Holger Rodriguez"]
+maintainers = ["Holger Rodriguez <github@roseng.ch>"]
+maintainers-logins = ["hgrodriguez"]
+tags = ["embedded", "nostd", "eeprom", "rp2040", "i2c"]
+website = "https://github.com/hgrodriguez/eeprom_i2c"
+
+[[depends-on]]  # Added by alr
+hal = "~0.1.0"  # Added by alr
```
Comment (mosteo) [inline index/ee/eeprom_i2c/eeprom_i2c-4.0.0.toml]: Latest version of `hal` is `0.3`, not sure if using this old one is in purpose?

Comment (hgrodriguez) [reply index/ee/eeprom_i2c/eeprom_i2c-4.0.0.toml]: you are right, i missed this, please delete this PR and I will fix this for hal 0.3.0, sorry for this.

**Comment by hgrodriguez** [comment]
@mosteo I see that you deleted 4.0.0, could 4.0.1 still be merged or do i have to raise another one?

**Comment by mosteo** [comment]
I was waiting for tests to finish, I'll merge this one.

**Review by mosteo — APPROVED** [review]

**Pull request merged**